Sources/Contentful/TypedQuery.swift

```diff
@@ -63,6 +63,35 @@ public final class LinkQuery<EntryType>: AbstractQuery where EntryType: EntryDec
         return LinkQuery<EntryType>.with(valueAtKeyPath: "fields.\(field.stringValue)", operation)
     }
 
+    /// Static method for creating a new LinkQuery with an operation. This variation for initializing guarantees
```

Comment:
Nitpick - better to use the multi-line comment style` /** ... */`

Reply:
Sure, will do. I just wanted to stick to what we have in the codebase right now. But you're right.